### PR TITLE
Add Checkov

### PR DESCRIPTION
**Pull request by nick-harmening**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by borchero** [comment]
Fine to be listed as maintainer ✅

File: recipes/checkov/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/checkov-{{ version }}.tar.gz
+  sha256: a968ca91e6ffe53d93f2c9b45c168096b4d0e7511b1e74d126c7aab68afc06cb
+
+build:
+  skip: true  # [py<38]
```
Comment (xhochy) [inline recipes/checkov/meta.yaml:13]:
```suggestion
```

File: recipes/checkov/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    - click >=8.0.0
+    - aiohttp
+    - aiodns
+    - aiomultiprocess
+    - jsonschema <5.0.0,>=4.6.0
+    - prettytable >=3.0.0
+    - pycep-parser ==0.4.1
+    - charset-normalizer
+    - pyston-autoload ==2.3.5  # [py<311 and ( linux or darwin )]
+    - pyston ==2.3.5  # [py<311 and ( linux or darwin )]
```
Comment (xhochy) [inline recipes/checkov/meta.yaml:62]:
```suggestion
    - pyston-autoload ==2.3.5  # [py<311 and unix]
    - pyston ==2.3.5  # [py<311 and unix]
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by xhochy** [comment]
Depends on https://github.com/conda-forge/pyston_lite-feedstock/pull/4

File: recipes/checkov/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/checkov-{{ version }}.tar.gz
+  sha256: a968ca91e6ffe53d93f2c9b45c168096b4d0e7511b1e74d126c7aab68afc06cb
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py<39 or (py<311 and osx and x86)]
```
Comment (xhochy) [inline recipes/checkov/meta.yaml:15]:
```suggestion
  skip: true  # [py<39 or (py<311 and osx and x86_64)]
```

File: recipes/checkov/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    - packageurl-python
+    - click >=8.0.0
+    - aiohttp
+    - aiodns
+    - aiomultiprocess
+    - jsonschema <5.0.0,>=4.6.0
+    - prettytable >=3.0.0
+    - pycep-parser ==0.4.1
+    - charset-normalizer
+    - pyston_lite_autoload ==2.3.5  # [py<311 and unix and x86]
```
Comment (xhochy) [inline recipes/checkov/meta.yaml:61]:
```suggestion
    - pyston_lite_autoload ==2.3.5  # [py<311 and unix and x86_64]
```

File: recipes/checkov/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    - pyston_lite ==2.3.5  # [py<311 and unix and x86]
```
Comment (xhochy) [inline recipes/checkov/meta.yaml:62]:
```suggestion
    - pyston_lite ==2.3.5  # [py<311 and unix and x86_64]
```

**Review by xhochy — CHANGES_REQUESTED** [review]
We prefer to use solely `x86_64`.

**Review by xhochy — APPROVED** [review]

**Pull request merged**